docs/_build/html/_static/jupyter_server_logo.svg

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Looks like we could do a quick round of cleanup in follow-up PRs, for example this file seem to be copied from the Jupyter Server team compass.

Reply:
Deleted this directory—this was supposed to be ignored. Thanks!